spec/unit/api_client_spec.rb

```diff
@@ -1,6 +1,27 @@
 require 'spec_helper'
 
 describe Typekitter::ApiClient do
+  describe '#kits_create' do
+    context 'require to typekit api fails' do
+      it 'returns a parsed string of error messages' do
+        json_response = "{\"errors\":[\"failed\"]}"
+        parsed_errors = "An error occured: #{json_response}"
+        fake_token = double("token", value: 'fake token 1234')
+        stub_request(:post, "https://typekit.com/api/v1/json/kits").
+          with(:headers => {
+          'User-Agent'=>'Typhoeus - https://github.com/typhoeus/typhoeus',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Surrounding space missing for operator '=>'.